board/src/main/java/com/hanhae99/board/repository/BoardRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.hanhae99.board.repository;
+
+import com.hanhae99.board.domain.Board;
+import com.hanhae99.board.domain.User;
+import jakarta.persistence.EntityManager;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+@RequiredArgsConstructor
+public class BoardRepository {
+
+    private final EntityManager em;
+
+    public List<Board> findAll(){
+        List<Board> result = em.createQuery("select b from Board b", Board.class)
```

Comment:
```suggestion
        List<Board> result = em.createQuery("select b from boards b", Board.class)
```

entity에 명시된 테이블 명과 다른 것 같아요.